scripts/go-to-slide.js

```diff
@@ -7,8 +7,10 @@ H5P.CoursePresentation.GoToSlide = (function ($) {
    * @class
    * @param {Number} slideNum
    * @param {CoursePresentation} cp
+   * @param {String} goToSlideNextPrevious
+   * @param {Number} index
    */
-  function GoToSlide(title, slideNum, invisible, cp) {
+  function GoToSlide(title, slideNum, invisible, cp, goToSlideNextPrevious, index) {
```

Comment:
_use cp.currentSlideIndex instead of index_
cp.currentSlideIndex is set to 0 in go-to-slide.js, so I couldn't use it. It looks like it specifies the slide that the end user has navigated to.

_rename goToSlideNextPrevious as goToSlideType_
Makes sense to me. I've added a new commit.

_Give the new option a default in the CP "constructor" function._
I followed the code back, but I got lost in the addElement & addElements functions. I was able to add this in the GoToSlide function though. Thoughts?